frontend/app/views/spree/checkout/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1 @@
+Put here your terms and conditions
```

Comment:
Not a big deal at all, but "Put your terms and conditions here" would probably be the more standard english way to say this?